ci-operator/config/openshift/release/openshift-release-master.yaml

```diff
@@ -17,6 +17,11 @@ tests:
   commands: make check-services
   container:
     from: src
+- as: ipi-deprovision-aws
+  cron: 45 4,10,16,22 * * *
+  steps:
+    cluster_profile: aws
+    workflow: ipi-deprovision-pruner-aws
```

Comment:
Can I set a single ref?  Maybe `ref: ipi-deprovision-pruner-aws`?  Or do I need to inline a workflow here?  Or...?

Reply:
[rehearsal][1]:

```
time="2020-12-09T22:04:55Z" level=error msg="Failed to get resolved config for test" error="failed resolve ReleaseBuildConfiguration: Failed resolve MultiStageTestConfiguration: no workflow named ipi-deprovision-pruner-aws" org=openshift repo=release
```

I've tried to resolve with 55c8767b8d -> 8fbd9b45d9:

```diff
-    workflow: ipi-deprovision-pruner-aws
+    test:
+    - ref: ipi-deprovision-pruner-aws
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14226/pull-ci-openshift-release-master-pj-rehearse/1336793690055643136#1:build-log.txt%3A18

Reply:
[Progress][1] (I think?):

```
2020/12/09 23:21:30 step <*steps.leaseStep> is missing dependencies: <&api.internalImageStreamTagLink{name:"release", tag:"latest", unsatisfiableError:""}>
2020/12/09 23:21:30 Ran for 0s
error: some steps failed:
  * could not print execution order: could not sort nodes: steps are missing dependencies
```

[1]:https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14226/rehearse-14226-periodic-ci-openshift-release-master-ipi-deprovision-aws/1336813179161808896#1:build-log.txt%3A5

Reply:
8fbd9b45d9 -> d2933c0083 moves the config from `openshift-release-master.yaml` to `openshift-release-master__ocp-4.8.yaml` to get some `releases` entries, so ci-operator can find a `release:latest`.

Reply:
[No luck][1]:

```
error: some steps failed:
  * failed to generate steps from config: failed to resolve release latest: failed to request latest release: server responded with 404: &{%!s(*http.body=&{0xc0003ae0c0 <nil> <nil> false false {0 0} false false false <nil>}) {%!s(int32=0) %!s(uint32=0)} %!s(bool=false) <nil> %!s(func(error) error=0x839c00) %!s(func() error=0x839b80)}
time="2020-12-09T23:53:36Z" level=info msg="Reporting job state 'failed' with reason 'defaulting_config:resolving_release'"
```

Despite:

```console
$ grep -A1 releases: ci-operator/config/openshift/release/openshift-release-master__ocp-4.8.yaml
releases:
  latest:
```

I dunno :shrug:

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14226/rehearse-14226-periodic-ci-openshift-release-master-ocp-4.8-ipi-deprovision-aws/1336821281818939392#1:build-log.txt%3A4